javalin/src/test/java/io/javalin/TestCompression.kt

```diff
@@ -143,8 +143,8 @@ class TestCompression {
     }
 
     @Test
+    @EnabledIf("brotliAvailable")
```

Comment:
How does this work? 🤔

Reply:
It's a function declared at the end of the file basically, if we use assumptions an exception is throw and the test failed for me on `aarm64`.

Reply:
I see. Not sure why this has to be sorted through strings, but I checked the docs, and it seems correct 🤔